processors/project-reference-labeler.ts

```diff
@@ -16,38 +18,111 @@ export function getDefaultConfiguration(): any {
   return <any>{
     'process-with-label': 'feature',
     'column-label-prefix': '> ',
-    'linked-label-prefix': '>> '
+    'linked-label-prefix': '>> ',
+    'label-color': '#FFFFFF',
+    // need to actually set to true, otherwise it's just a preview of what it would write
+    'write-labels': false
   }
 }
 
-const noiseWords = ['the', 'in', 'and', 'of']
+// const noiseWords = ['the', 'in', 'and', 'of']
 
 function cleanLabelName(prefix: string, title: string) {
   title = title.replace(/\([^()]*\)/g, '').replace(/ *\[[^\]]*]/, '')
 
-  let words = title.match(/[a-zA-Z0-9]+/g)
+  const words = title.match(/[a-zA-Z0-9&]+/g)
   //  words = words.map(item => item.toLowerCase())
 
-  words = words.filter(word => noiseWords.indexOf(word) < 0)
+  //words = words.filter(word => noiseWords.indexOf(word) < 0)
```

Comment:
epic labels are from column names.  However, feature label links are created from titles that users create who are not consumers of this tool.  Titles can be super long.  Considered tidying it up with just words and no noise words so it's not super long but still distinct and discernable